doc/tutorials/platformer/step_5.md

```diff
@@ -308,6 +308,9 @@ if (position.x + 64 >= game.size.x / 2 && horizontalDirection > 0) {
   velocity.x = 0;
   game.objectSpeed = -moveSpeed;
 }
+
+position += velocity * dt;
+super.update(dt);
```

Comment:
I don't think the super.update is needed here if I remember this tutorial correctly?

Reply:
![image](https://github.com/flame-engine/flame/assets/29718978/f66290d0-88bd-4c12-b180-4a7a88c3eb13)

It's there, but not entirely sure whether it's actually needed or not

Reply:
I had a look, it actually is needed. Thanks for your contribution!